client/src/components/Header/Header.js

```diff
@@ -17,12 +17,16 @@ export default class Header extends React.Component {
                 <div className="left">
                     <div className="links">
                         {this.links.map((link, index) =>
-                            <NavLink to={link.route} exact={link.exact} className="header-link" activeClassName="active" key={link.name}>{link.name}</NavLink>
+                            <NavLink to={link.route} exact={link.exact} className="header-link mr-4" activeClassName="active" key={link.name}>{link.name}</NavLink>
                         )}
                     </div>
                 </div>
                 <div className="center">
-                    Put a search bar here pls
+                    <Form inline>
+                        <FormControl type="text" placeholder="Title" size="sm" className="mr-sm-2" />
+                        <FormControl type="text" placeholder="Location" size="sm" className="mr-sm-2" />
```

Comment:
Maybe the placeholder for this search bar could be "Near..." or something along those lines.